ffi/capi/bindings/dart/CalendarKind.g.dart

```diff
@@ -87,11 +87,11 @@ enum CalendarKind {
         return 10;
       case hebrew:
         return 11;
-      case hijriTabularFridayEpochTypeIi:
+      case hijriTabularTypeIiFriday:
         return 12;
       case hijriSimulatedMecca:
         return 18;
-      case hijriTabularThursdayEpochTypeIi:
+      case hijriTabularTypeIiThursday:
```

Comment:
thoughts on using `Type2`?

Reply:
Tbh I'd just use II. Rust casing tradition is XmlHttpRequest, but tons of people do it the other way.

Not opposed to numbers though.

Reply:
Oh, this is Dart. Type2 is good.

Might also be possible to tweak the casing thingy.

Reply:
we use `TypeII` in Rust, but this goes through `heck` so in Dart it becomes `TypeIi`